src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/CreateVideoBookingService.kt

```diff
@@ -39,6 +40,8 @@ class CreateVideoBookingService(
     }
 
   private fun createCourt(request: CreateVideoBookingRequest, createdBy: User): Pair<VideoBooking, Prisoner> {
+    checkCaseLoadAccess(request.prisoner().prisonCode!!)
```

Comment:
Only needed for courts bookings on create as prison users cannot create probation bookings.